### ISSUE TITLE
SQL assistant/Completion will work with synonyms ?

### ISSUE THREAD

**Issue by vipinaugstine** [dbeaver/dbeaver#5208]
#### System information: 
- Operating system (distribution) and version : windows 10 Enterprise
- DBeaver version : 5.2.3
- Java version  : 8 update 151
- Additional extensions

#### Connection specification:
- Database name and version :Oracle  11g
- Driver name ; ojdbc6
- Do you use tunnels or proxies (SSH, SOCKS, etc)? 

#### Describe the problem you're observing:
         
 I'm using oracle 11g database and all my tables are distributed in different schema and I have created synonyms to access it. 
Whether SQL Editor assertive completion feature will work with these synonyms? 
It is working properly with tables residing in same schema and 'schema.tables' from different as well. but when come with synonym assertive completion is not working. 
I can't use 'schema.table' in procedure/queries to enable this feature, so kindly advise me how assertive completion can be enable for table synonym which is pointing to a table in different schema of same oracle database. 

Thanks.
vpn

**Comment by serge-rider** [comment]
Search in synonyms was broken after #5208 fix. 
Reverting with additional fixes.

**Comment by p91paul** [comment]
Did you mean #5036? Can you implement something less drastic than removing the option to exclude synonyms? They make searches way slower, so they are a very interesting object type to remove when you don't need them.
If you think it's doable, I'm happy to open a separate issue.

**Comment by vipinaugstine** [comment]
my question was different, whether sql editor assistant completion will work if we try to query a  synonym which is  referring to a table in different scheme. ?
if i use owner_schema.table then sql assistant is showing all the columns resides in the table but if I create a synonym for owner_schema.table and try to access with sysnonym  name sql assistant is not fetching the column name.

please advise.

**Comment by vipinaugstine** [comment]
![issue](https://user-images.githubusercontent.com/47348916/52324704-67656200-29f3-11e9-8f10-d42a36fc9b3b.jpg)

Is there any settings, database object sharing option to get this? My system is already designed in this way so removing synonym is not an option now. I just want to know any option to set auto completion for synonym which will help user to write the code in more easy way. Attached screenshot.

**Comment by p91paul** [comment]
Oh, nice, you are right! This is an entirely different issue. Looking at @serge-rider commit, he got confused too! Nevermind, I opened #5221 to track the entirely different issue I was experiencing.

**Comment by vipinaugstine** [comment]
Any option to solve this problem? how I will get a solution, Im very new here
